decayers/ERElasticScattering.cxx

```diff
@@ -200,6 +217,14 @@ Bool_t ERElasticScattering::Stepping() {
       LOG(DEBUG) << "  Lab out2 state(px,py,pz,E) = "<<out2V.Px()<<","<<out2V.Py()<<","<<out2V.Pz()
                 << "," << out2V.E() << FairLogger::endl;
 
+      /* DEBUG*/
+      static Int_t nEvent = 0;
+      if (nEvent == 0) {
+        std::cerr << "[ Detector = " << setw(2) << fThetaRangeCenter << " ]:";
+        std::cerr << " thetaCM = " << generatedTheta*RadToDeg() << ", thetaLab = " << setw(9) << out2V.Theta()*RadToDeg();
```

Comment:
Move it to Init method. And you will not need this static staff.

Reply:
I think, we don't need this part. I am going to remove It.